WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormRemoteActionUseCase.swift

```diff
@@ -73,13 +73,14 @@ final class ProductFormRemoteActionUseCase {
         addProduct(product: productModelToSave,
                    password: password,
                    successEventName: successEventName,
-                   failureEventName: failureEventName) { result in
+                   failureEventName: failureEventName) { [weak self] result in
+            guard let self else { return }
             switch result {
             case .success(let data):
-                guard data.product.productType == .variable else {
+                let variableTypes: [ProductType] = [.variable, .variableSubscription]
```

Comment:
Nice work updating this to duplicate variations of the subscription product. I didn't even know that we had a duplicate feature. 👏

Reply:
> I didn't even know that we had a duplicate feature. 

I [added](https://github.com/woocommerce/woocommerce-ios/pull/7727) the feature and you approved it! 😆